Gemfile

```diff
@@ -5,6 +5,7 @@ gemspec
 gem "rake"
 gem "rails"
 gem "redis-client"
+# gem "debug"
```

Comment:
suggestion: as `Gemfile.lock` is not checked into git repo, maybe introduce trick like `Gemfile.local` to avoid this kind of commits
ref: https://github.com/rubocop/rubocop/blob/bd42ff683f2da12a01c829aaac95c0575de1f129/Gemfile#L30